packages/web/src/js/FileUploader.ts

```diff
@@ -0,0 +1,192 @@
+      if (process.env.NODE_ENV === 'development') {
+        console.warn('Unfortunately, there is no list element created for inserting files to upload ⚠️');
+      }
+
+      return;
+    }
+
+    if (!this.isMultiple) {
+      this.clearFileQueue();
+    }
+
+    // Creating an element to insert into a list
+    const id = file.name.replace(/\.[^/.]+$/, '');
+    const listElementNode = document.createElement('li');
+    listElementNode.setAttribute('id', id);
+    listElementNode.classList.add('FileUploaderAttachment');
+    listElementNode.innerHTML = `
+      <svg width="24" height="24" aria-hidden="true">
+        <use xlink:href="/icons/svg/sprite.svg#file" />
+      </svg>
+      <span class="text-truncate">${file.name}</span>
+      <button type="button" class="FileUploaderAttachment__remove" data-dismiss="${id}">
+        <span class="accessibility-hidden">Remove</span>
+        <svg width="24" height="24" aria-hidden="true">
+          <use xlink:href="/icons/svg/sprite.svg#close" />
+        </svg>
+      </button>
+    `;
+
+    // Appending node into list
+    this.listElement.appendChild(listElementNode);
+
+    // Appending an item into queue
+    this.fileQueue?.push(file);
+
+    // Adding an event for the delete item button
+    const currentItem = SelectorEngine.findOne(`[data-dismiss="${id}"]`);
+    currentItem?.addEventListener('click', (event) => {
+      event.preventDefault();
+      const target = (event.target as HTMLElement)?.dataset?.dismiss;
+      if (target) {
+        this.removeFromQueue(target);
+      }
+    });
+
+    // Reset drag state
+    this.isDragging = false;
+    this.wrapper?.classList.remove(IS_DRAGGING_CLASS_NAME);
+  }
+
+  addToQueue(file: File) {
+    const allowedFileSize = file.size <= FILE_SIZE_LIMIT;
+    const duplicity = this.fileQueue.find((item) => item.name === file.name);
+    const run = this.fileQueue.length === 0 ? true : this.fileQueue.length >= 1 && !duplicity;
+
+    if (run && allowedFileSize) {
+      this.fileQueue?.push(file);
+      this.appendToList(file);
+    } else if (!allowedFileSize) {
+      console.warn('The file size limit has been exceeded ⚠️');
+    } else {
+      console.warn('This file already exists in the queue ⚠️');
+    }
+  }
+
+  removeFromQueue(name: string) {
+    const index = this.fileQueue.findIndex((item) => item.name.replace(/\.[^/.]+$/, '') === name);
+    if (index > -1) {
+      const element = SelectorEngine.findOne(`li#${name}`);
+      const currentItem = SelectorEngine.findOne(`[data-dismiss="${name}"]`);
+      this.fileQueue.splice(index, 1);
+      currentItem?.removeEventListener('click', () => null);
+      element?.remove();
+    }
+  }
+
+  onChange(event: Event) {
+    const { files } = event.target as HTMLInputElement;
+    const filesArray = files ? [...files] : [];
+
+    filesArray.forEach((file) => this.addToQueue(file));
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  onDragOver(event: Event) {
```

Comment:
```suggestion
  static onDragOver(event: Event) {
```

and you can remove the comment above.

Reply:
This is something else entirely. I don't need to make the function static, I'm just making an exception for not using this in the method. If I make the method static, I completely change its meaning and then I'm forced to do more modifications.

Reply:
done